common/types/mapper/thrift/admin.go

```diff
@@ -1051,8 +1051,8 @@ func FromResendReplicationTasksRequest(t *types.ResendReplicationTasksRequest) *
 	}
 	return &admin.ResendReplicationTasksRequest{
 		DomainID:      t.DomainID,
-		WorkflowID:    t.WorkflowID,
-		RunID:         t.RunID,
+		WorkflowID:    &t.WorkflowID,
+		RunID:         &t.RunID,
```

Comment:
there are only a few of these "nullable pointer -> non-null pointer"-containing files, but I think I need to read them more carefully.  it seems like there's a decent chance they're correct/safe, but it's definitely a semantic change.

my brain needs a bit of a break after reading everything else though, so I'll come back to this later :)

Reply:
admin's good - everything that uses this clearly requires the fields